web/src/App.tsx

```diff
@@ -56,7 +56,7 @@ const App: React.FC = () => {
                             <SignOut />
                         </Route>
                         <Route path={FirstFactorRoute}>
-                            <LoginPortal />
+                            <LoginPortal rememberMe={configuration?.remember_me_enabled}/>
```

Comment:
```suggestion
                            <LoginPortal rememberMe={configuration?.remember_me_enabled === true}/>
```
And then you can remove ` | undefined` in every props types.

Reply:
The less type unions, the better.